pygmt/datasets/samples.py

```diff
@@ -123,3 +123,26 @@ def load_fractures_compilation():
     fname = which("@fractures_06.txt", download="c")
     data = pd.read_csv(fname, header=None, sep=r"\s+", names=["azimuth", "length"])
     return data[["length", "azimuth"]]
+
+
+def load_hotspots():
+    """
+    Load a table with the locations, names, and suggested icon sizes of
+    hotspots.
+
+    This is the ``@hotspots.txt`` dataset used in the GMT tutorials.
+
+    The data are downloaded to a cache directory (usually ``~/.gmt/cache``) the
+    first time you invoke this function. Afterwards, it will load the data from
+    the cache. So you'll need an internet connection the first time around.
+
+    Returns
+    -------
+    data : pandas.DataFrame
+        The data table with columns "longitude", "latitude", "icon_size", and
+        "placename".
+    """
+    fname = which("@hotspots.txt", download="c")
+    columns = ["longitude", "latitude", "icon_size", "placename"]
```

Comment:
```suggestion
    columns = ["longitude", "latitude", "symbol_size", "placename"]
```
I recommend symbol_size since symbol is used elsewhere in the docs more than icon.